highlight/highlight
# Fix antlr crashes

## Summary
There is an edge case where the frontend crashes when typing in an `AND` after an open parenthesis. Fix the type error.

https://www.loom.com/share/474a6d9cb7bd4af2922cd64b347d02cd?sid=5bda8150-6ae1-4748-8dfc-322a70aaf8a9

## How did you test this change?
1) Open the logs or traces search
2) Start searching for `source=(backend AND frontend)`
- [ ] Page does not crash
- [ ] No results should be shown

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
N/A
